START: Loading latest COVID-19 data
Errors while scraping website and loading to JSON and CSV:
Traceback (most recent call last):
  File "app.py", line 1, in <module>
    import parse
  File "/Users/home/Documents/Coding/Projects/covid19-newjersey/python/parse.py", line 70
SyntaxError: Non-ASCII character '\xe2' in file /Users/home/Documents/Coding/Projects/covid19-newjersey/python/parse.py on line 70, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
Commiting changes to github:
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
fatal: could not read Username for 'https://github.com': Device not configured
FINISH: Completed data update for COVID-19 in NJ